src/pretalx/agenda/templates/agenda/schedule_list_days.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n %}
+{% load static %}
+
+<div class="schedule-as-list">
+    {% for day in data %}
+        <h3 id="{{ day.start.date|date:"c" }}">
+            <a href="#{{ day.start.date|date:"c" }}">{{ day.start.date|date:"DATE_FORMAT" }}</a>
+        </h3>
+        <div class="day">
+            {% if day.first_start %}
+                <div class="talk-container talks-list">
+                    {% for talk in day.talks %}
+                        {% if not schedule.is_archived %}
+                         <a href="{{ talk.submission.urls.public }}">
+                        {% else %}
+                         <a>
+                        {% endif %}
+                            <div class="talk{% if request.user in talk.submission.speakers.all %} talk-personal{% endif %}{% if talk.is_active %} active{% endif %}{% if search %} {% if search in talk.submission.title.lower or search in talk.submission.display_speaker_names.lower %} search-hit{% else %} search-fail{% endif %}{% endif %}"
+                                 id="{{ talk.submission.code }}"
+                                 title="{{ talk.submission.title }} {% if talk.submission.speakers.exists %}({{ talk.submission.display_speaker_names }}){% endif %}"
+                                 style="{% if request.event.settings.use_tracks and talk.submission.track %} border-color: {{ talk.submission.track.color }}{% endif %}"
+                                 data-time="{{ talk.start|date:"H:i" }}–{{ talk.end|date:"H:i" }}">
+                                <div class="talk-content">
+                                    {% if talk.submission.do_not_record %}
+                                        <span class="fa-stack">
+                                          <i class="fa fa-video-camera fa-stack-1x"></i>
+                                          <i class="fa fa-ban do-not-record fa-stack-2x" aria-hidden="true" title="{{ phrases.agenda.schedule_do_not_record }}"></i>
+                                        </span>
+                                    {% endif %}
+
+                                    {% if talk.submission.is_deleted %}
```

Comment:
This replicates a lot of content – if possible, it would be better to include this content from a shared template.